pkg/controller/volume/persistentvolume/scheduler_binder_cache.go

```diff
@@ -71,6 +75,10 @@ func (c *podBindingCache) UpdateBindings(pod *v1.Pod, node string, bindings []*b
 		nodeBinding = nodeBindings{}
 		c.bindings[podName] = nodeBinding
 	}
+	if _, ok := nodeBinding[node]; !ok {
+		metrics.VolumeBindingAddToSchedulerBinderCache.Inc()
```

Comment:
I think this can just be updated in the block above, since the metrics is based on the pod name, not node.